### ISSUE TITLE
Create executables in a separate folder called ./bin

### ISSUE THREAD

**Issue by jithin-k-sreedharan** [krzysztof-turowski/duplication-divergence#38]
- Otherwise .gitignore file should contain all the executable names. 
- If this is implemented, just add bin/ in .gitignore, and remove the current list of explicit executable names from .gitignore

**Comment by krzysztof-turowski** [comment]
Right, in general we should at some point move sources to src/ folder and exectuables to build/ or bin/

As for now I added to .gitignore to omit all files without extension, this should solve the problem temporarily.
